### ISSUE TITLE
Push code to a newly created scratch org that has Field History enabled for an object

### ISSUE THREAD

**Issue by renatoliveira** [forcedotcom/salesforcedx-vscode#501]
### Summary
Enabled the Dev Hub in a customer production org. Then I proceeded to create a package to retrieve the necessary metadata, and then created a new scratch org with the following definition:

```
{
    "orgName": "Company",
    "edition": "Developer",
    "orgPreferences" : {
        "enabled": ["S1DesktopEnabled", "IsKnowledgeEnabled"]
    },
    "features": [
        "PersonAccounts",
        "ServiceCloud",
        "AuthorApex"
    ]
}
```

When using `sfdx force:source:push -u ProdCopy3` I get a lot of deployment errors, one in particular is related to the field history not being enabled on certain opportunity fields, like the following:

```The entity: Opportunity does not have history tracking enabled (998:13)```

Now, according to the [Scratch Org Definition Configuration Values](https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_scratch_orgs_def_file_config_values.htm) it is possible to specify history for contracts only (searching on that page for "history" returns only that result), but there is no sign of the history property for other objects.

### Steps To Reproduce:
1. Retrieve metadata from an org with field history enabled on some object on a package.
1. Push said metadata to a newly created scratch org.
1. Observe the "history tracking not enabled" error.

### Expected result
A property to explicitly specify the objects with history tracking enabled, or history tracking being enabled automatically for objects when specified in the field's metadata file.

### Actual result
Deployment errors.

**VS Code Version**: 1.24.1

**SFDX CLI Version**: sfdx-cli/6.20.1-120ba3807b (darwin-x64) node-v8.9.4

**OS and version**: macOS 10.13.5 (High Sierra)

**Comment by renatoliveira** [comment]
Thank you for your reply @ntotten.

I only realised this issue isn't exactly related to the VS Code extension after posting it. This is one of the cases where I wish GitHub would allow us to delete issues/comments, haha.

If it helps, I believe this is related to [this issue](https://success.salesforce.com/issues_view?id=a1p3A0000018BX2QAM&title=sfdx-force-source-push-command-fails-on-history-tracking) on the Known Issues website (this link too I got only a few days after opening this issue).
